Packs/ExtFilter/Scripts/ExtFilter/README.md

```diff
@@ -8049,6 +8049,7 @@ If `if` condition is not given or returns any value, `then` operation is execute
 | if | expressions | (Optional) `if` condition |
 | then | expressions | Conditions to execute if `if` condition is not given or returns any value. |
 | else | expressions | (Optional) Conditions to execute if `if` returns `null`. |
+| lhs | expressions | (Optional) The value used on the left hand side instead of using the current element. |

```

Comment:
Not sure I understood what the 'lhs' does ... Can you provide a more detailed or clearer explanation?

 In addition, you can change line 8048 with the following:
 | --- | --- | --- |
 that will create an MD table (you can change that in many places in the readme file if you like).